src/kubelet/group.go

```diff
@@ -82,6 +83,32 @@ func (r *kubelet) Group(definition.SpecGroups) (definition.RawGroups, *data.Erro
 		}
 	}
 
+	// Convert node conditions to a map so our ValueFuncs can work nicely with them
```

Comment:
Dots could be consistent.
```suggestion
	// Convert node conditions to a map so our ValueFuncs can work nicely with them.
```